src/WebJobs.Script/ManagedDependencies/IManagedDependencyConfigs.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
+
+namespace Microsoft.Azure.WebJobs.Script.ManagedDependencies
+{
+    internal interface IManagedDependencyConfigs
+    {
+        string RapidUpdateAzModulePath { get; }
```

Comment:
What is this property for?

Reply:
The latest Az module is copied on the windows OS as part of the Rapid update at this path for optimization purpose, so that when the customer's app is dependent upon latest version of the Az module, so it won't require download, this is for optimization in module download process. This path gets added to PSModulePath for the PS language worker to add it for loading.

Reply:
We should have language specific properties here.  This should go into powershell worker config?

Reply:
The latest Az module is copied on the windows OS as part of the Rapid update at this path for optimization purpose, so that when the customer's app is dependent upon latest version of the Az module, so it won't require download, this is for optimization in module download process. This path gets added to PSModulePath for the PS language worker to add it for loading.

Reply:
PS Language worker needs to handle this logic i.e if Az module is already available do not download

Reply:
Had a conversation with Anirudh on this and since host is deciding if an module needs to be downloaded or not so it would need host to check if the requested Az module is already available or not